mygnupg.py

```diff
@@ -0,0 +1,64 @@
+import gnupg
+
+class mygnupg():
+    """
+    This class used for initial gnupg, 
+    create, import, and export key pairs.
+    Also, can encrypt and decrypt data.
+    """
+    def __init__(self):
+        """
+        The binary file should be different
+        in different os.
+        This is MacOS version.
+        """
+        self.gpg = gnupg.GPG(gnupghome='./gnupgkeys')
+        self.key = None
+        self.email = "john_smith@scmail.org"
+        self.keytype = "RSA"
+        self.password = "sample"
+
+    def import_key(self, path):
```

Comment:
Since this raises an error if the key doesn't exist there's no chance to create a key when I'm running this code.